.travis.yml

```diff
@@ -26,6 +27,9 @@ matrix:
           dist: trusty
           env: T=normal
         - os: linux
+          compiler: gcc
+          env: T=gssapi
```

Comment:
Yeah, I'll try that, thanks!
I also want to try out the other approach suggested few comments above as a separate PR, so we can look at both approaches.